lib/identity/domain/use_cases/profile/check_profile_validity_use_case.dart

```diff
@@ -17,9 +17,11 @@ class CheckProfileValidityUseCase
     extends FutureUseCase<CheckProfileValidityParam, void> {
   @override
   Future<void> execute({required CheckProfileValidityParam param}) {
+    final maxVal = BigInt.parse('2') ^ BigInt.parse('248');
     return Future(() {
-      if (param.profileNonce < 0 ||
-          (param.profileNonce == 0 && param.excludeGenesis)) {
+      if (param.profileNonce.isNegative ||
+          (param.profileNonce == BigInt.zero && param.excludeGenesis) ||
```

Comment:
Should it be `param.profileNonce == GENESIS_PROFILE_NONCE`?

Reply:
resolved